File: src/main/java/com/znl/event/QuartzJob.java
package com.znl.event;


import org.apache.commons.lang.StringUtils;
import org.quartz.Job;
import org.quartz.JobExecutionContext;
import org.quartz.JobExecutionException;

/**
 * 定时job
 */
public class QuartzJob implements Job {

    public void execute(JobExecutionContext jobExecutionContext) throws JobExecutionException {
        /*JobDetail jobDetail=jobExecutionContext.getJobDetail();
        String jobName = ((JobDetailImpl)jobDetail).getName();   //任务名称
        System.err.println(jobName);*/
       // System.err.println(jobExecutionContext.getTrigger().getKey().getName());
        String cmdName=jobExecutionContext.getTrigger().getKey().getName();
        if(StringUtils.isBlank(cmdName))return;
        FixedTimeEventHandler.executeFixedTimeEvent(Integer.parseInt(cmdName));
    }

}

File: src/main/scala/com/znl/event/QuartzJob.scala
package com.znl.event

import org.apache.commons.lang.StringUtils
import org.quartz.{Job, JobExecutionContext}

/**
 * Created by Administrator on 2016/6/25.
 */
class QuartzJob extends Job{
  def execute(jobExecutionContext:JobExecutionContext ){
    /*JobDetail jobDetail=jobExecutionContext.getJobDetail();
    String jobName = ((JobDetailImpl)jobDetail).getName();   //任务名称
    System.err.println(jobName);*/
    // System.err.println(jobExecutionContext.getTrigger().getKey().getName());
    val cmdName:String =jobExecutionContext.getTrigger().getKey().getName();
    System.err.println(cmdName)
    if(StringUtils.isBlank(cmdName))return;
    FixedTimeEventHandler.executeFixedTimeEvent(Integer.parseInt(cmdName));
  }
}
